BUILD.md

```diff
@@ -249,4 +249,18 @@ Overall coverage rate:
 
 The report will be generated in HTML form to alow visualization of each line of code coverage for each C++ file in the project. The entry HTML page to the report is located at the C++ build folder: `cpp/build/coverage/index.html`
 
-In VSCode, line by line coverage can be visualized using the extension [gcov-viewer](https://marketplace.visualstudio.com/items?itemName=JacquesLucke.gcov-viewer)
\ No newline at end of file
+In VSCode, line by line coverage can be visualized using the extension [gcov-viewer](https://marketplace.visualstudio.com/items?itemName=JacquesLucke.gcov-viewer)
+
+
+## Fuzzing
+
+Go code can be easily fuzzed: fuzzing attempts to find bugs by mutating a test input data set and measuring test coverage. This complementary technique aims stability issues.
+
+### Fuzzing Evm.StepN interface
+
+The ct Evm.StepN interface is used to evaluate one N instructions in different EVM implementations. There are 2 Fuzzer tests against this interface:
+
+- Crash test: FuzzLfvm will execute instructions one at the time looking for a panic. ```make fuzz-lfvm```
+- Differential test: FuzzDifferentialLfvmVsGeth ```make fuzz-lfvm-diff```
+
+Notice lack of evmzero target: go utilizes native code coverage to inform the algorithm, which would use this information to drive the mutation process. There is currently no method to retrieve this information from the C++ library, this renders the C++/go fuzzer as a brute-force algorithm. 
```

Comment:
Running it in `brute-force` mode might be better than nothing, right?

Reply:
Actually writing this sentence got me thinking: Running the diff between geth and evmzero, will retrieve coverage reports from geth, and blindly stress similar regions in evmzero. I will make a couple of runs today and add a target for this purpose.

Reply:
evmzero differential fuzzer yielded results, just with the seed corpus. Those need to be fixed before we can even start the fuzzing: 
[#549](https://github.com/Fantom-foundation/Tosca/issues/549)